ocs_ci/ocs/utils.py

```diff
@@ -790,6 +790,21 @@ def setup_ceph_toolbox(force_setup=False):
             rook_toolbox = OCS(**toolbox)
             rook_toolbox.create()
             return
+
```

Comment:
We should create an issue to revert this workaround. We should also add a comment in here stating that this is a workaround and link to the created issue so that it will be addressed once the workaround is no longer necessary.

Reply:
Done.